apps/addons/decorators.py

```diff
@@ -10,6 +10,12 @@
 log = commonware.log.getLogger('mkt.purchase')
 
 
+def owner_or_unlisted_reviewer(request, addon):
+    return (acl.check_unlisted_addons_reviewer(request) or
+            acl.check_addon_ownership(request, addon, admin=False, dev=True,
```

Comment:
Ok, I naively thought admins were superusers. So that makes sense, maybe add a small comment.